pkg/model/model.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package model;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+message Traces {
+  repeated Trace traces = 1 [(gogoproto.nullable) = false];
+}
+
+message Trace {
+  uint64 trace_id = 1;
+  repeated Span spans = 2 [(gogoproto.nullable) = false];
+}
+
+message Span {
```

Comment:
Thats a good point - with this change we're no longer zipkin compatible on the most part (the API will still be, as I don't want to implement my own UI)...

I've tried to make these protos more closely match the OpenTracing concepts & keywords.  I'll include an update to the readme in this PR.

Thanks for the feedback Adrian!